README.md

```diff
@@ -47,19 +47,18 @@
   </a>
 </p>
 
-In essence Vocabulary is a collection of UI components, which are parts of 
```

Comment:
These changes are inconsequential in practice but they are making what should be a very simple PR very big.